packages/bitcore-node/test/benchmark/wallet-benchmark.ts

```diff
@@ -11,11 +11,11 @@ async function getAllAddressesFromBlocks(start, end) {
   return Object.keys(addresses.reduce((prev, a) => Object.assign(prev, { [a.address]: a.address }, {})));
 }
 
-async function createWallet(addresses: string[], iteration) {
+export async function createWallet(addresses: string[], iteration) {
   const walletName = 'Benchmark Wallet' + iteration;
   const password = 'iamsatoshi';
   const chain = 'BTC';
-  const network = 'mainnet';
+  const network = 'regtest';
```

Comment:
This will break the benchmark script, perhaps pass it in as an argument to this function